File: C/EX34.c
/*programa que identifica qual o maior numero digitado*/
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char const *argv[])
{
    //declaração de variáveis e do vetor
    int num;
    int vetor[10];

    printf("Digite 10 numeros.");

    //preenchimento do vetor
    for (int i = 0; i < 10; i++)
    {
        printf("\nn: ");
        scanf("%i", &vetor[i]);
    }
    
    //inicialização da variável 'num' com o primeiro valor do vetor, no caso 'vetor[0]'
    num = vetor[0];


    //identificação do maior número
    for (int i = 0; i < 10 ; i++)
    {
        if (num > vetor[i]) 
        {
            num;
        }
        
        else
        {
            num = vetor[i];
        }
        
    }
    
    printf("\nO maior numero digitado: %i", num);

    return 0;
}

File: Python/EX34.py
#acha o maior valor dentre os digitados
numeros=[]

for i in range(0, 10, 1):
    x=int(input("N%d: " %(i+1)))
    numeros.append(x)

print(f"vetor: {numeros}")

num = numeros[0]

for u in range(0,10,1):
    
    if num > numeros[u]:
        num
    else:
        num = numeros[u]

print(f"Maior número digitado: {num}")




'''
numeros = list(range(10))

x=0

for i in range(1,11,1):
    numeros[x]=int(input("N%d: " % (x+1)))
    x+=1

print(f"Vetor: {numeros}")

num = numeros[0]

for u in range(0,10,1):
    
    if num > numeros[u]:
        num
    else:
        num = numeros[u]

print(f"Maior número digitado: {num}")
'''